src/states_screens/race_gui_base.cpp

```diff
@@ -1039,6 +1039,21 @@ void RaceGUIBase::drawPlayerIcon(AbstractKart *kart, int x, int y, int w)
                                                       NULL, true);
         }
     }
+
+	//lap flag for finished karts
+	if (kart->hasFinishedRace())
+	{
+        video::ITexture *icon_lapflag = irr_driver->getTexture(FileManager::GUI_ICON, "lap_flag.png");
+        if (icon_lapflag != NULL)
+		{
+            const core::rect<s32> rect(core::position2d<s32>(0, 0),
+                                       icon_lapflag->getSize());
+            const core::rect<s32> pos1(x - 20, y - 10, x + w - 20, y + w - 10);
```

Comment:
many hardcoded values here, have you tested varied resolutions?

Reply:
Same as above, I just copied from the attachments here so I suspect I should work if it works for the attachments.